srcpkgs/kindd/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'kindd'
+pkgname=kindd
+version=2.0.1
+revision=1
+wrksrc="Kindd-${version}"
+build_style=qmake
+hostmakedepends="git qt5-qmake"
+makedepends="qt5-devel qt5-declarative-devel qt5-quickcontrols2-devel"
+depends="polkit-gnome qt5-quickcontrols qt5-graphicaleffects qt5-quickcontrols2 qt5-declarative qt5-svg"
+short_desc="Kindful dd gui written in qt quick"
+maintainer="linarcx <linarcx@gmail.com>"
+license="GPL-3.0-or-later"
```

Comment:
GPL-3.0-only

Reply:
why?

Reply:
From LICENSE:
> Each version is given a distinguishing version number.  If the
Program specifies that a certain numbered version of the GNU General
Public License "or any later version" applies to it, you have the
option of following the terms and conditions either of that numbered
version or of any later version published by the Free Software
Foundation.  If the Program does not specify a version number of the
GNU General Public License, you may choose any version ever published
by the Free Software Foundation.

LICENSE is GPL-3.0 so it specifies version, but I can't find permission for later ones.